src/components/ParticipantListItem.js

```diff
@@ -97,47 +107,32 @@ const ParticipantName = styled.Text `
   text-transform: capitalize;
 `;
 
-const ButtonsContainer = styled.View `
-  display: flex;
-  flex-direction: row;
-  margin-top: ${theme.SPACING_SMALL}
-`;
-
 const TeamFormationStatus = styled.View `
   width: 10;
   height: 10;
   border-radius: 5;
   background: ${props => props.statusColor};
 `;
 
-const ButtonContainer = styled.TouchableOpacity `
+const ButtonsContainer = styled.View `
   display: flex;
   flex-direction: row;
-  margin-right: 5;
-`;
-
-const AcceptTextContainer = styled.View `
-  padding-horizontal: 10;
-  padding-vertical: 5;
-  border-top-left-radius: 100;
-  border-bottom-left-radius: 100;
-  border-top-right-radius: 100;
-  border-bottom-right-radius: 100;
-  background-color: ${theme.COLOR_GREEN};
-  width: 72;
-  align-items: center;
+  margin-right: ${theme.SPACING_SMALL}};
 `;
 
-const DenyTextContainer = styled(AcceptTextContainer) `
-  background-color: ${theme.COLOR_RED};
+const ButtonContainer = styled.TouchableOpacity `
+  display: flex;
+  flex-direction: row;
+  margin-right: ${theme.SPACING_XSMALL}};
 `;
 
-const ButtonText = styled.Text`
-  color: white;
-  font-family: ${theme.FONT_SEMIBOLD};
-  letter-spacing: ${theme.LETTER_SPACING_SMALL};
-  text-transform: uppercase
-  font-size: ${theme.FONT_SIZE_MEDIUM};
-`;
+const AcceptButton = styled.Image `
+  height: 20;
```

Comment:
I love the look of the button icons, however they're a bit tiny. Perhaps you can add the buttons to the next row and make them a bit larger?

<img src="https://user-images.githubusercontent.com/43888017/159096343-0dc09c93-ed0e-45f8-86ff-cc60c109aee3.png" width="300" />

Reply:
Stella didn’t like the original one to spill onto the next line :( so we opted for these icons.

Reply:
I changed the size of the buttons, see if it's better!